ivy_tests/test_ivy/helpers/function_testing.py

```diff
@@ -125,6 +125,9 @@ def test_function(
         returned, or whether the flattened numpy arrays are returned.
     all_as_kwargs_np
         input arguments to the function as keyword arguments.
+    backend_to_test
+        The backend (framework) to test the function against, e.g., "torch", "numpy",
+        etc.
```

Comment:
```suggestion
      The backend framework to be tested.
```

Let's not include examples in the docstring for parameter. Let's update the other docstring similarly, Thanks!